### DISCUSSION TITLE
3월 4주차 초안 글

### DISCUSSION THREAD

**Discussion by stopmin** [BDD-CLUB/challenge-every-week-posting#395] [4. 기타 글 모음]
안녕하세요 :D

이번주에는 kotlin에서 코루틴을 활용하여 비동기 처리를 하는 부분에 관련한 글을 작성하고 있습니다!! 그런데 일이 아직 끝나지 않아서 글 완성은 못시켰습니닷 ㅠㅠ 이번주 내로 완성하고 회고와 함께 올리도록 하겠습니다 !! 

- https://stopmin.tistory.com/entry/KotlinSpring%EC%9C%BC%EB%A1%9C-Kinesis-Consumer-Producer-%EB%B9%84%EB%8F%99%EA%B8%B0-%EC%B2%98%EB%A6%AC%EC%9E%91%EC%84%B1%EC%A4%91

하핫 그리고 약간의 TMI를 말하자면, 이번주에 코테 & 자소서 이슈로 (카테캠) 나름 정신이 없었습니닷... 
그리고 블로그도 벨로그에서 티스토리로 옮기게 되었습니다!! 

벨로그에 여러번 요청을 날리니깐 404에러도 뜨고, 커스텀 할 수 없다는 것이 너무 아쉬워서 1학년 때 티스토리를 쓰다가 편의를 위해 벨로그를 쓰다가 다시 티스토리로 옮겨가게 되었습니닷 ㅎㅎ 다들 이번주도 파이팅 합시다!! 다음주에는 알고리즘 글도 꼭꼭꼭꼭 하나는 쓰겠습니다아...

https://stopmin.tistory.com/

**Comment by 1jeongg** [comment]
오 카테캠 지원하셨나보네요!! 버디님이시라면 분명 좋은 결과 있을거예요 (●'◡'●)

작년에 저도 백엔드 카테캠 했었는데 혹시 도움 필요하시면 언제든 말씀해주세요!

**Reply by stopmin** [reply]
고맙습니다 :)

**Comment by shkisme** [comment]
넘나 멋져요 👍

**Reply by stopmin** [reply]
아닙니다!
